vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -319,16 +312,43 @@ type::ExternalWalletPtr ResetWallet(type::ExternalWalletPtr wallet) {
   wallet = type::ExternalWallet::New();
   wallet->type = constant::kWalletUphold;
 
-  if (status != type::WalletStatus::NOT_CONNECTED) {
-    if (status == type::WalletStatus::VERIFIED) {
-      wallet->status = type::WalletStatus::DISCONNECTED_VERIFIED;
-    } else {
-      wallet->status = type::WalletStatus::DISCONNECTED_NOT_VERIFIED;
-    }
+  if (status == type::WalletStatus::VERIFIED) {
+    wallet->status = type::WalletStatus::DISCONNECTED_VERIFIED;
   }
 
   return wallet;
 }
 
+template <typename T, typename... Ts>
+bool one_of(T&& t, Ts&&... ts) {
+  bool match{};
+
+  static_cast<void>(std::initializer_list<bool>{
+      (match = match || std::forward<T>(t) == std::forward<Ts>(ts))...});
```

Comment:
Not sure what the `match = match` is doing?

Reply:
It's a simple assignment (`match = match || std::forward<T>(t) == std::forward<Ts>(ts)`), mimicking the compound assignment operator for `||` (`match ||= std::forward<T>(t) == std::forward<Ts>(ts)`, which doesn't exist).

Reply:
Got it, thanks!